README.md

```diff
@@ -54,6 +54,16 @@ The JSON control file has the format:
 
 It's possible it could do more than just filter in the future.
 
+`deco` can currently source it's JSON control file from:
```

Comment:
`its` :P

Reply:
hey!  I thought i was the grammar nanny!